### PR TITLE
Converted final TGO Module test

### LINKED ISSUE #4666: Get Clean Jenkins Dev Tests
**ISIS version(s) affected**: dev

**Description**  

Several test are failing regularly and haven't been addressed. These have been disabled and we need to fix them in order to get clean tests. 

These include: 
 1. edrget app test failures
 2. TGO cat test failures
 3. ObervationPair fixture failing on setup

**How to reproduce**  

Revert changes in https://github.com/USGS-Astrogeology/ISIS3/pull/4664 and run gtest

**Possible Solution**  

- [ ] remove edrget application, tests and documentation (#4665)
- [ ] Somehow fix the kernels used by the TGO tests, TGO category tests seem to fail because of changing kernels  (#4661)
- [x] fix ObservationPair segfaults (context #4609)

### PR DESCRIPTION
**Pull request by acpaquette**
Removes final tgo cassis test and associated makefiles

## Description
Updates the last Tgo Cassis module test to use the new gtest framework.

## Related Issue
Closes #4666 

## Motivation and Context
Clean jenkins tests

## How Has This Been Tested?
This is a test

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Documentation change (update to the documentation; no code change)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have read and agree to abide by the [Code of Conduct](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/Code-Of-Conduct.md)
- [x] I have read the [**CONTRIBUTING**](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/CONTRIBUTING.md) document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.
- [x] I have added myself to the [.zenodo.json](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/.zenodo.json) document.
- [x] I have added any user impacting changes to the [CHANGELOG.md](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/CHANGELOG.md) document.

## Licensing
This project is mostly composed of free and unencumbered software released into the public domain, and we are unlikely to accept contributions that are not also released into the public domain. Somewhere near the top of each file should have these words:

> This work is free and unencumbered software released into the public domain. In jurisdictions that recognize copyright laws, the author or authors of this software dedicate any and all copyright interest in the software to the public domain.

- [x] I dedicate any and all copyright interest in this software to the public domain. I make this dedication for the benefit of the public at large and to the detriment of my heirs and successors. I intend this dedication to be an overt act of relinquishment in perpetuity of all present and future rights to this software under copyright law.

### CODE REVIEW

File: isis/tests/TgoCassisModuleTests.cpp [hunk 1]
```diff
@@ -2178,3 +2178,593 @@ TEST(TgoCassisModuleTests, TgoCassisUncontrolledSingleColorMosaic) {
...
+                                 "to=" + prefix.path() + "/exported.img"};
+  UserInterface rdrgen(RDRGEN_XML, rdrgenArgs);
+  try {
+    tgocassisrdrgen(rdrgen);
+  }
+  catch (IException &e) {
+    FAIL() << "Unable to run tgocassisrdrgen on projected image: " << e.what() << std::endl;
+  }
+
+  // QString ingestedExportFile = prefix.path() + "/exported.cub";
```
Comment (AustinSanders) [inline isis/tests/TgoCassisModuleTests.cpp:2344]: Can we remove the commented code or is there a reason to keep it around?

**Review by AustinSanders — COMMENTED** [review]
Overall looks good. Just one question about commented out code, but should be ready for merge

File: isis/tests/TgoCassisModuleTests.cpp (hunk 1 above)
Comment (acpaquette) [reply isis/tests/TgoCassisModuleTests.cpp:2344]: We need to be able to check that the output from tgocassisrdrgen is valid. This is commented out as we originally thought the output product could just be ingested but this turn out to be incorrect.

I will put in an issue to get this tested properly. Setting up the infrastructure to test this properly is outside the scope of support hours

**Review by AustinSanders — DISMISSED** [review]

**Comment by jlaura** [comment]
@acpaquette This has been approved for a while - what is the status?

**Comment by acpaquette** [comment]
@jlaura Failing tests that I have not been able to get to yet. I plan on finishing this PR this coming week as Autocnet concludes

**Review by AustinSanders — APPROVED** [review]

**Pull request merged**